autogen/docs/codetab.md.mustache

```diff
@@ -5,6 +5,17 @@ to use immediately go in the Command Center; commands you want to save and use
 later, over and over again, are found in the Code tab.
 ![[codetab.gif]]
 
+The Code tab can be opened in a separate window from the main NetLogo window. It can then be moved and resized.
+
+### Ways to separate and rejoin the Code tab
+- The "Code Tab in separate window" checkbox can be used to switch between the joined (part of the main NetLogo window) and separated states.
+- A Control-click with the mouse on the tab bar of the Code tab will also switch its state.
+- To separate a joined Code tab you can use Command + [ on Mac, Control + [
+number on other operating systems.
+- To rejoin a separated Code tab you can use Command + ] on Mac, Control + ]
+number on other operating systems.
+- To rejoin a separated Code tab you can click the close button.
+
```

Comment:
It seems very odd for this section to come before the table of contents.  I'd recommend putting it after the "Checking for Errors" section, and giving it its own two-`#` header so it also shows up in the ToC.  If you do that, I think you can probably drop the "Ways to separate and rejoin..." header.